### ISSUE TITLE
Correlation between rows in the two datasets

### ISSUE THREAD

**Issue by TimmermanIDS** [RUG-IDS/Course-2017#35]
Hi,

We were wondering if it is true that a certain row in the 6d data corresponds to the same row in the 3d data. In other words, if we look at the same row in the 6d data and the 3d data, do we have the data of a single star, or two different ones?

Thanks in advance.

Cheers,
Team 9

**Comment by rm-bwana** [comment]
Shall keep this open just in case fellow students wish to see.

I have confirmed that a row in the 6d data and 3d data are referring to the same star. You can try to use the extra information to help with the clustering, but we do not guarantee that this will make clustering easier. You can test whether combining these actually make a difference in the clustering performance.

One thing to note is that the 3d information are answers of calculations performed on the 6d information in combination of information neighbouring stars. This may result in strong correlation between features that may not be necessary. 

TL;DR: You can try combine the 2 files if you wish, don't expect the clouds to part and the sun to shine on a correct answer if you do.
